Automattic/vip-go-mu-plugins
# Disable indexing of filtered post_content by default

## Description

Filtered post content is not searched by default (and is of little value in the index), so skip indexing it by default.

## Checklist

Please make sure the items below have been covered before requesting a review:

- n/a This change has relevant documentation additions / updates (if applicable).

## Steps to Test

1. Check out PR.
1. `wp shell`
1. Then with a valid post id, re-index it then check to ensure `post_content_filtered` is an empty string:
1. `\ElasticPress\Indexables::factory()->get( 'post' )->index( 1 );`
1. `\ElasticPress\Indexables::factory()->get( 'post' )->get( 1 );`
